### ISSUE TITLE
Script error

### ISSUE THREAD

**Issue by tunavis** [tigattack/VeeamNotify#53]
### What happened?

![image](https://user-images.githubusercontent.com/8057000/187655736-88dafd34-40dd-4b4e-a078-c856087ba34e.png)

Receiving the follow error in veeam. Nothing in the logs file. 

### VeeamNotify version

v1.0

### PowerShell version

5.1

### Veeam Backup & Replication build number

11.0.1.1261

### Relevant log output

```PowerShell
Log folder empty
```

**Comment by tigattack** [comment]
Could you please share your job log? This can be found at `%ProgramData%\Veeam\Backup\<JOBNAME>\Job.<JOBNAME>.Backup.log`.

Please remember to scrub any sensitive information.

I'm interested in the log output from the line stating `Running post-job script` onwards.

**Comment by sopleb** [comment]
I'm running into this same issue... 

```
Running post-job script
[14.11.2022 10:17:27] <01> Info         Executing custom command [C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe], arguments [-NoProfile -ExecutionPolicy Bypass -File 'C:\VeeamScripts\VeeamNotify\Bootstrap.ps1']
[14.11.2022 10:17:27] <01> Warning      Post-job script terminated with exit code -196608
```

**Comment by sopleb** [comment]
Interesting... I removed the single quotes from the command and it worked fine...

```
[17.11.2022 00:04:47] <01> Info         Executing custom command [Powershell.exe], arguments [-File C:\VeeamScripts\VeeamNotify\Bootstrap.ps1]
[17.11.2022 00:04:57] <01> Info         Post-job script completed successfully
```

**Comment by tigattack** [comment]
Good catch @sopleb, thanks! That's interesting, I've not seen anything like that before.

Since I'm unable to reproduce the original issue, would you mind trying it again with the `-NoProfile -ExecutionPolicy Bypass` parameters and the full path to `powershell.exe`, but with no quotes on the `Bootstrap.ps1` path?

It would also be useful to know what version of Veeam B&R you're running.

**Comment by sopleb** [comment]
> Good catch @sopleb, thanks! That's interesting, I've not seen anything like that before.
> 
> Since I'm unable to reproduce the original issue, would you mind trying it again with the `-NoProfile -ExecutionPolicy Bypass` parameters and the full path to `powershell.exe`, but with no quotes on the `Bootstrap.ps1` path?
> 
> It would also be useful to know what version of Veeam B&R you're running.
https://i.imgur.com/4ubYIHT.png


```
[18.11.2022 11:00:36] <01> Info         Running post-job script
[18.11.2022 11:00:36] <01> Info         Executing custom command [C:\Windows\system32\WindowsPowerShell\v1.0\powershell.exe], arguments [-NoProfile -ExecutionPolicy Bypass -File C:\VeeamScripts\VeeamNotify\Bootstrap.ps1]
[18.11.2022 11:01:08] <01> Info         Post-job script completed successfully
```

**Comment by GoryMadLogger** [comment]
I'd noticed a similar issue when setting mine up around the ' I endedup with this
`C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe -NoProfile -ExecutionPolicy Bypass -File C:\VeeamScripts\VeeamNotify\Bootstrap.ps1`
This is on Veeam 12
